core/trino-main/src/test/java/io/trino/execution/TestCommentTask.java

```diff
@@ -63,7 +64,7 @@ public void testCommentTableOnView()
 
         assertTrinoExceptionThrownBy(() -> getFutureValue(setComment(TABLE, asQualifiedName(viewName), Optional.of("new comment"))))
                 .hasErrorCode(TABLE_NOT_FOUND)
-                .hasMessage("Table '%1$s' does not exist, but a view with that name exists. Did you mean COMMENT ON VIEW %1$s IS ...?", viewName);
+                .hasMessageContaining("Table '%1$s' does not exist, but a view with that name exists. Did you mean COMMENT ON VIEW %1$s IS ...?", viewName);
```

Comment:
oh, I see it now, this now contains a location?

Reply:
Right, the message contains a dummy location at the beginning.